eureka/lib/astropytable.py

```diff
@@ -35,6 +35,8 @@ def savetable_S3(filename, bjdtdb, wave_1d, stdspec, stdvar, optspec, opterr):
     opterr = opterr.flatten()
 
     arr = [bjdtdb, wave_1d, stdspec, stdvar, optspec, opterr]
+    arr_shape = [len(a) for a in arr]
```

Comment:
Why was this added?

Reply:
This is me debugging things and not tidying up properly, I'll remove it.